src/box/lua/config/annotations.lua

```diff
@@ -0,0 +1,2100 @@
+    using the audit_log.to option. If you write logs to a file, Tarantool
+    reopens the audit log at SIGHUP.
+]])
+
+M['audit_log.filter'] = 'Enable logging for a specified subset of audit events.'
+
+-- ?
+M['audit_log.filter.*'] = toline([[
+    Specify a subset of audit events to log by providing a value
+    from the allowed list of events or groups.
+]])
+
+M['audit_log.format'] = 'Specify a format that is used for the audit log.'
+
+M['audit_log.nonblock'] = toline([[
+    Specify the logging behavior if the system is not ready to write.
+    If set to `true`, Tarantool does not block during logging if the system
+    is non-writable and writes a message instead. Using this value may
+    improve logging performance at the cost of losing some log messages.
+]])
+
+M['audit_log.pipe'] = toline([[
+    Specify a pipe for the audit log destination. You can set the `pipe` type
+    using the audit_log.to option. If log is a program, its pid is stored in
+    the `audit.pid` field. You need to send it a signal to rotate logs.
+]])
+
+M['audit_log.spaces'] = toline([[
+    The array of space names for which data operation events (`space_select`,
+    `space_insert`, `space_replace`, `space_delete`) should be logged. The array
+    accepts string values. If set to box.NULL, the data operation events are
+    logged for all spaces.
+]])
+
+-- ?
+M['audit_log.spaces.*'] = 'Space name'
+
+M['audit_log.to'] = 'Enable audit logging and define the log location.'
+
+M['audit_log.syslog'] = toline([[
+    This module allows configuring the system logger (syslog) for audit
+    logs in Tarantool. It provides options for specifying the syslog server,
+    facility, and identity for logging messages.
+]])
+
+M['audit_log.syslog.facility'] = toline([[
+    Specify a system logger keyword that tells syslogd where to send the
+    message. You can enable logging to a system logger using the
+    `audit_log.to` option.
+]])
+
+M['audit_log.syslog.identity'] = toline([[
+    Specify an application name to show in logs. You can enable logging
+    to a system logger using the `audit_log.to` option.
+]])
+
+M['audit_log.syslog.server'] = toline([[
+    Set a location for the syslog server. It can be a Unix socket path
+    starting with "unix:" or an ipv4 port number. You can enable logging
+    to a system logger using the `audit_log.to` option.
+]])
+
+-- }}} audit_log configuration
+
+-- {{{ compat configuration
+
+M['compat'] = 'The compat section defines values of the compat module options.'
+
+M['compat.binary_data_decoding'] = toline([[
+    Define how to store binary data fields in Lua after decoding:
+    `new`: as varbinary objects,
+    `old`: as plain strings.
+]])
+
+M['compat.box_cfg_replication_sync_timeout'] = toline([[
+    Set a default replication sync timeout:
+    `new`: 0,
+    `old`: 300 seconds.
+]])
+
+-- DM
+M['compat.box_consider_system_spaces_synchronous'] = '?'
```

Comment:
Some fields are not in the [documentation](https://www.tarantool.io/en/doc/latest/reference/configuration/configuration_reference/), they are marked `--DM`

Reply:
If a description is correct and helpful, I don't care about its initial source.

I guess you have doubts regarding some of the descriptions and want to highlight an attention on them. I propose the following process.

* You do your best to provide correct and useful descriptions. Use the website documentation, the documentation requests from https://github.com/tarantool/doc/issues, commit messages, pull request descriptions, information from the relevant issues. If you have doubts, you can ask authors of relevant commits or me.
* If there are still some descriptions that need a special attention, leave a pull request comment -- this way we'll discuss them and update as part of the review process.

The final version of the patchset shouldn't have these `DM`/`SM` marks, I think.

Reply:
> The final version of the patchset shouldn't have these `DM`/`SM` marks, I think.

Sure. I removed these marks because the missing descriptions are now filled from other sources. I think I can close this conversation.